.vsconfig

```diff
@@ -0,0 +1,6 @@
+﻿{
```

Comment:
Could you just quickly add this file to gitignore and then delete it? Let's avoid adding trash files if possible